_data/simple-icons.json

```diff
@@ -2788,6 +2788,11 @@
             "hex": "C10C0C",
             "source": "https://en.wikipedia.org/wiki/File:Cairo_metro_logo2012.svg"
         },
+        {
+            "title": "CaixaBank",
+            "hex": "007EAE",
+            "source": "https://www.caixabank.es/deployedfiles/particulares/Estaticos/Imagenes/Geolocalizador_CXB.svg"
```

Comment:
The site is down. I can't verify the source at this time.

Reply:
Looks like the site is back up

Reply:
https://downforeveryoneorjustme.com/caixabank.es?www=1

![](https://github.com/user-attachments/assets/099219d0-e8c4-49de-9ea8-496d4a4cd651)

I still cannot visit the site.

Reply:
Odd, I am still able to access the site just fine
![Screenshot 2024-12-09 at 3 38 51 PM](https://github.com/user-attachments/assets/cea7abc3-e995-4f6f-9124-85606594c67b)

Reply:
I still can not open the site. I've tried many country IPs through the VPN, but no luck.